app/src/main/java/org/liberty/android/fantastischmemo/common/AnyMemoDBOpenHelper.java

```diff
@@ -80,8 +80,8 @@ private void convertOldDatabase(SQLiteDatabase database) {
         // This is old database
         if (isOldDatabase) {
             // copy all cards
-            database.execSQL("insert into cards (ordinal, question, answer, note, hint)" +
-                    " select _id as ordinal, question, answer, note, hint from dict_tbl");
+            database.execSQL("insert into cards (ordinal, question, answer, note)" +
+                    " select _id as ordinal, question, answer, note from dict_tbl");
```

Comment:
Did you remove the hint because we need to keep the original schema initially and then update it only after the tables have been created?

Reply:
yes exactly